openqa-label-known-issues

```diff
@@ -134,14 +134,16 @@ investigate_issue() {
     # against even in case when autoinst-log.txt is missing the details, e.g.
     # see https://progress.opensuse.org/issues/69178
     echo "$reason" >> "$out"
-    if [[ "$curl_output" != "200" ]] && [[ "$curl_output" != "301" ]]; then
+    # Control flow when there is a curl issue but still want to process job due to reason var
+    # even if autoinst-log.txt not found but assume is missing.
+    [[ -n $reason ]] && [[ $curl_output = "404" ]] && [[ $reason != null ]] \
+        && is_unreachachle_but_has_reason=1 \
+        || is_unreachachle_but_has_reason=0
+    if [[ "$curl_output" != "200" ]] && [[ "$curl_output" != "301" ]] && [[ ! $is_unreachachle_but_has_reason ]]; then
         # if we can not even access the page it is something more critical
         handle_unreachable "$testurl" "$out"
-        # Assume autoinst-log.txt still not found but process job anyway due to reason var
-        if [[ -n $reason ]] && [[ $curl_output = "404" ]] && [[ $reason != null ]]; then
-            label_on_issues_from_issue_tracker "$id"
```

Comment:
So this is no longer called?

Reply:
it is not called here but it does run, when `no autoinst-log.txt but reason exist`. I found it weird that we had to handle that case in this block. I believe the approach now, it simplifies it a bit.